opensrp-reveal/src/test/java/org/smartregister/reveal/shadow/RevealMapViewShadow.java

```diff
@@ -0,0 +1,23 @@
+package org.smartregister.reveal.shadow;
+
+import android.content.Context;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.util.AttributeSet;
+
+import org.robolectric.annotation.Implementation;
+import org.robolectric.annotation.Implements;
+import org.smartregister.reveal.view.RevealMapView;
+
+/**
+ * Created by samuelgithengi on 1/23/20.
+ */
+@Implements(RevealMapView.class)
+public class RevealMapViewShadow extends KujakuMapViewShadow {
+
+    @Implementation
+    public void __constructor__(@NonNull Context context, @Nullable AttributeSet attrs) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The instance method name '__constructor__' doesn't match '[a-z][a-zA-Z0-9]*'](https://app.codacy.com/manual/OpenSRP/opensrp-client-reveal/pullRequest?prid=4684568)